MH4U Dex/ItemContainerViewController.m

```diff
@@ -0,0 +1,61 @@
+//
+//  ItemContainerViewController.m
+//  MH4U Dex
+//
+//  Created by Joseph Goldberg on 2/19/15.
+//  Copyright (c) 2015 Joseph Goldberg. All rights reserved.
+//
+
+#import "ItemContainerViewController.h"
+
+#import "ItemMonsterSourcesTableViewController.h"
+#import "ItemAreaSourcesTableViewController.h"
+
+@interface ItemContainerViewController ()
+
+@property (weak, nonatomic) IBOutlet UIView *monsterSourceSubView;
+@property (weak, nonatomic) IBOutlet UIView *areaSourceSubView;
+@property (weak, nonatomic) IBOutlet UILabel *itemNameLabel;
+
+@end
+
+@implementation ItemContainerViewController
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    [self.areaSourceSubView setHidden:YES];
+    self.navigationItem.title = [NSString stringWithFormat:@"%@ Drop Sources", self.itemName];
+    self.itemNameLabel.text = self.itemName;
+}
+
+- (IBAction)segmentedControlChanged:(UISegmentedControl *)segmentedControl
+{
+    switch (segmentedControl.selectedSegmentIndex) {
+        case 0:
+            [self.monsterSourceSubView setHidden:NO];
```

Comment:
[The fourth bullet here](https://github.com/vokal/Coding-Standards/tree/master/iOS#implementation-styling) says, in part:

> Use dot notation for properties

[The sixth bullet here](https://github.com/vokal/Coding-Standards/tree/master/iOS#method-styling) says:

> Use dot syntax for accessing properties and only for accessing properties